pandas/computation/tests/test_eval.py

```diff
@@ -50,11 +50,7 @@ def _eval_single_bin(lhs, cmp1, rhs, engine):
         try:
             return c(lhs, rhs)
         except ValueError as e:
-            try:
-                msg = e.message
-            except AttributeError:
-                msg = e
-            msg = u(msg)
+            msg = u(str(e))
```

Comment:
I am fixing a bunch of little issues in #13010 . so just remove from here.

``DeprecationWarnings`` I DO care about. want to get rid of them.